software/fishstick-stm32f401/.openocd/stlink-blackpill.cfg

```diff
@@ -0,0 +1,7 @@
+source [find interface/stlink.cfg]
```

Comment:
perhaps we should just call this `stlink.cfg` or something unrelated to the `blackpill`. I was planning on doing this for the stm32f1xx config as well.

Reply:
filename changed